src/sentry/replays/endpoints/organization_replay_count.py

```diff
@@ -57,7 +76,21 @@ class OrganizationReplayCountEndpoint(OrganizationEventsV2EndpointBase):
         }
     }
 
+    @extend_schema(
+        operation_id="Return a count of replays for a given issue or transaction id.",
```

Comment:
```suggestion
        operation_id="Return a Count of Replays",
```

i believe this is the sidebar title, which are all capitalized and fairly short/concise on the existing docs